tailwind.config.js

```diff
@@ -1,4 +1,4 @@
-const plugin = require('tailwindcss/plugin')
+import plugin from 'tailwindcss/plugin';
```

Comment:
I'll have a quick look whether this changes the required nodejs version, after that I'll approve the PR.

Reply:
does not seem like it